ci-operator/step-registry/redhat-chaos/pod-scenarios/random-system-pods/redhat-chaos-pod-scenarios-random-system-pods-workflow.yaml

```diff
@@ -0,0 +1,13 @@
+workflow:
+  as: redhat-chaos-pod-scenarios-random-system-pods
+  steps:
+    test:
+      - ref: redhat-chaos-pod-scenarios-random-system-pods
+    env:
+      TARGET_NAMESPACE: "openshift-.*"
+      DISRUPTION_COUNT: "1"
+      EXPECTED_POD_COUNT: "3"
```

Comment:
there won't be 3 pods here... not good that this passed.

Reply:
I don't see this getting filled into the scenario config 

```
- id: kill-pods
  config:
    namespace_pattern: openshift-*
    name_pattern: .*
    kill: 1
    timeout: 180
```

Reply:
think this is ok because we have an if statement to get around this... thoughts on taking out the values because it could be confusing?
https://github.com/redhat-chaos/krkn-hub/blob/a593ccdd826ba4985bf892ad8697eea3170c8393/prow/pod-scenarios/prow_run.sh#L24-L28

Reply:
Makes sense! Updated. Thanks.